tutorials/01_01_introduction.ipynb

```diff
@@ -80,15 +84,26 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
+   "id": "8d197413",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
I wonder how output still ends up being committed, if we have this precommit hook to clean the notebooks?

Reply:
eventually we should create our own little local pre-commit script that does what we want :)

Reply:
well, I also ran the clean notebooks script, but it obviously didn't do the trick. Since there are no relevant changes to this file, I can simply copy the original version from master into this branch. Anyways, shouldn't be important for this PR.